tests/nnfw_api/src/one_op_tests/Quantize.cc

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "GenModelTest.h"
+
+#include <memory>
+
+CircleGen genSimpleQuantizeModel(circle::TensorType from_t, circle::TensorType to_t)
+{
+  CircleGen cgen;
+  int in = cgen.addTensor({{1, 4, 4, 1}, from_t}, 1, 128);
+  int out = cgen.addTensor({{1, 4, 4, 1}, to_t}, 1, 0);
```

Comment:
How about using changed scale from input?

Reply:
If output type is `INT8_ASYMM`, it would be nice to change zero_point to a non-zero number.

Reply:
@ragmani 

> How about using changed scale from input?

Could you explain more what you mean? I think the scale is same `1` after `Quantize` u8 to i8.

Reply:
> If output type is `INT8_ASYMM`, it would be nice to change zero_point to a non-zero number.

@ragmani It would take more time since I made the test case by myself. Do you have any testcase that I could use for complex test case?

Reply:
> Could you explain more what you mean? I think the scale is same 1 after Quantize u8 to i8.

The `Quantize` operation can be change scale value from input to output. So it would be nice to change scale from input to output to for more strict verification.

Reply:
> > Could you explain more what you mean? I think the scale is same 1 after Quantize u8 to i8.
> 
> The `Quantize` operation can change scale value from input to output. So it would be nice to change scale from input to output to for more strict verification.

Okay. You want to change output scale to different scale. It took me much more time but consequently it gets better test case.